src/main/java/io/anserini/search/SearchWebCollection.java

```diff
@@ -240,38 +252,9 @@ public void search(SortedMap<Integer, String> topics, String submissionFile, Sim
     out.close();
   }
 
-  public void search(RerankerCascade cascade, SortedMap<Integer, String> topics, String submissionFile, Similarity similarity, int numHits, final String runTag) throws IOException {
```

Comment:
Oh the search method with that signature wasn't used anywhere, should I change it back?